equipment_handler/src/adcs_handler.c

```diff
@@ -578,8 +581,8 @@ ADCS_returnState ADCS_get_file_download_block_stat(bool *ready, bool *param_err,
     state = adcs_telemetry(DL_BLOCK_STAT_ID, telemetry, 5);
     *ready = telemetry[0] & 0x1;
     *param_err = telemetry[0] & 0x2;
-    *crc16_checksum = (telemetry[2] << 8) + telemetry[1];
-    *length = (telemetry[4] << 8) | telemetry[3];
+    *crc16_checksum = ((telemetry[2] << 8) & 0xFF00) | telemetry[1];
+    *length = ((telemetry[4] << 8) & 0xFF00) | telemetry[3];
```

Comment:
Why the bitwise AND? Bit shifting will result in zeros on the rightmost bits

Reply:
Not sure. Will remove

Reply:
Agreed that memcpys will cause problems. ADCS is little-endian

Reply:
As for the 0xFF00s, we'll remove them